tests/test_buku.py

```diff
@@ -562,6 +562,28 @@ def test_import_md(tmpdir, newtag, exp_res):
     res = list(import_md(p.strpath, newtag))
     assert res[0] == exp_res
 
+@pytest.mark.parametrize(
+    "newtag, exp_res",
+    [
+        (None, ("http://example.com", "text1", ",", None, 0, True, False)),
+        ("tag1", ("http://example.com", "text1", ",tag1,", None, 0, True, False)),
+    ],
+)
+def test_import_rss(tmpdir, newtag, exp_res):
+    from buku import import_rss
+
+    p = tmpdir.mkdir("importrss").join("test.rss")
+    p.write(
+        '<feed xmlns="http://www.w3.org/2005/Atom">\n'
+        '<title>Bookmarks</title>\n'
+        '<generator uri="https://github.com/jarun/buku">buku</generator>\n'
+        '    <entry>\n'
```

Comment:
…Shouldn't the `<title>` and `<generator>` tags be indented at the same level as the `<entry>` one? All three are children of the same `<feed>` node.

Reply:
Yes, good point. Will fix.